common-codegen/src/main/resources/strTemplate/objectType.stg

```diff
@@ -0,0 +1,119 @@
+template(input) ::=
```

Comment:
I really like the structure of our new java templates and think that they're easier to understand than our old javapoet based one 👍